### ISSUE TITLE
Old bugs

### ISSUE THREAD

**Issue by unleu** [tudorsss/Food-Tracker#85]
1. https://envoy.myjetbrains.com/youtrack/issue/HB-1354 - flagged - **Not fixed**

2. https://envoy.myjetbrains.com/youtrack/issue/HB-1369 

    Patients :  **Not fixed**
    Patient/Insurance:  **Not fixed**
    Patient/Medication:  **Not fixed**
    Treatment/Intake:  **Not fixed**

3 https://envoy.myjetbrains.com/youtrack/issue/HB-1420   **Not fixed**

4 https://envoy.myjetbrains.com/youtrack/issue/HB-1435 **Fixed**

5 https://envoy.myjetbrains.com/youtrack/issue/HB-1353 **partially fixed** only new users appear on the drop down list

6 https://envoy.myjetbrains.com/youtrack/issue/HB-1389  **Not fixed**

7 https://envoy.myjetbrains.com/youtrack/issue/HB-1397 I don't exactly know what to test. What is the tooltip they refer to ?

**Comment by tudorsss** [comment]
Hey @unleu,

For 1, if you take a look at my comment, I said that I should not test this ticket until another ticket is fixed (and that other ticket is not fixed yet).

For 2, I just talked with Chirag, he said he still has to work on it so he moved it out of Needs Review.

For 3, I know that the fix is not on our server because the bug was moved into Needs Review yesterday at 4:00 AM, but our server got updated on Monday, so I  flagged it to test it next week.

For 5, HB-1353 is fixed; this was found as a regression: https://envoy.myjetbrains.com/youtrack/issue/HB-1449. The old users not appearing is another bug, already added: https://envoy.myjetbrains.com/youtrack/issue/HB-1298

For 6, the same thing, the bug is fixed ,but HB-1289 is related to it.

For 7, google it when you don't know what it means: https://www.google.ro/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=what%27s%20a%20tool%20tip . I closed it, check the comments also.
